projects/ngx-d3timeline/src/lib/core/resize-observable.ts

```diff
@@ -0,0 +1,25 @@
+import { fromEventPattern } from 'rxjs';
+import { NodeEventHandler } from 'rxjs/internal/observable/fromEvent';
```

Comment:
probably shouldn't import something from `rxjs/internal`. If `NodeEventHandler` isn't part of the external interface maybe just use `any`?

Reply:
overlooked this, have removed the import and typed it accordingly.